Sources/TensorFlow/Core/LazyTensorShapeInference.swift

```diff
@@ -44,6 +44,8 @@ extension LazyTensorOperation {
     }
 
     func updateOutputShapes() {
+        outputShapes = Array<TensorShape?>(repeating: nil, count: outputCount)
```

Comment:
Redundant generic argument.
```swift
outputShapes = Array(repeating: nil, count: outputCount)
```

Reply:
@rxwei Sorry, I just noticed this comment. I will be reverting this change shortly anyway.